README.md

```diff
@@ -19,52 +19,65 @@ YourAppDelegate.h
 
 YourAppDelegate.m
 ```objective-c
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+    EDQueueStorageEngine *fmdbBasedStorage = [[EDQueueStorageEngine alloc] initWithName:@"database.sample.sqlite"];
```

Comment:
Interface-wise I'm not thrilled with introducing this additional complexity to the user as a requirement (though I think having a pluggable storage engine is a great feature addition). My preference would be to introduce this as an optional interface.